src/test/java/seedu/address/logic/commands/FindAttendanceCommandTest.java

```diff
@@ -0,0 +1,190 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_WEEK_1;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_WEEK_5;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_ITEM;
+import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD_ITEM;
+import static seedu.address.testutil.TypicalTutorsPet.getTypicalTutorsPet;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.attendance.Attendance;
+import seedu.address.model.attendance.Week;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.moduleclass.ModuleClass;
+import seedu.address.model.student.Student;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code FindAttendanceCommand}.
+ */
+public class FindAttendanceCommandTest {
```

Comment:
Perhaps you can consider writing a test case for finding attendance in a filtered list?